### PR TITLE
Added ACE Attach posibility to add more types of attachable items

### PR DESCRIPTION
**Pull request by kavinsky**
This adds a new config value: `ACE_attachable_effect` it contains the classname for the effect (vehicle)

### CODE REVIEW

**Comment by esteldunedain** [comment]
Thanks @kavinsky. I like the general idea of this, but there at least a remaining issue with the implementation:
- The `fnc_getChildrenAttachActions`, which has to create an action for each attachable item still uses the `"ACE_Attachable"` parameter.

File: addons/attach/CfgMagazines.hpp [hunk 1]
```diff
@@ -3,10 +3,32 @@ class CfgMagazines {
     class CA_Magazine;
     class B_IR_Grenade: CA_Magazine {
         ACE_Attachable = 1;
+        ACE_attachable_effect = "B_IRStrobe";
```
Comment (esteldunedain) [inline addons/attach/CfgMagazines.hpp]: To be consistent with other modules, this config etry should be renamed to something like: `ACE_attach_item`

**Comment by kavinsky** [comment]
yeah, i didnt remeve ACE_Attachable, just added new config. i can merge the two values.

File: addons/attach/stringtable.xml [hunk 2]
```diff
@@ -175,5 +103,29 @@
       <Russian>Присоединить Ошибка</Russian>
       <Spanish>Error al acoplar</Spanish>
     </Key>
+    <Key ID="STR_ACE_Attach_Item_Attached">
+      <English>%1 Attached</English>
```
Comment (esteldunedain) [inline addons/attach/stringtable.xml]: Insert a line break after the magazine name, as it probably won't be syntactically correct sentence for some languages:
`<English>%1&lt;br/&gt;Attached</English>`

**Pull request merged**